### PR TITLE
Feat: Multichain payment

### PR DESCRIPTION
**Pull request by x1m3**

### CODE REVIEW

**Review by martinsaporiti — DISMISSED** [review]
not sure about the name of the archives, specially payments_service... that file has methods for payment options. Besides that, the names of the methods ... like `createPaymentoption` should not be `create`?

**Comment by x1m3** [comment]
@martinsaporiti There are 2 concepts by now. Payments and PaymentOptions. Payments will be linked to PaymentOptions. I was in doubt of splitting this into 2 services: Payments and PaymentOptions , or to use just one. What do you think? Should we divide it into 2 services?

File: api/api.yaml [hunk 1]
```diff
@@ -1292,6 +1292,192 @@ paths:
...
+          description: Payment Request Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BasicMessage'
+        '400':
+          $ref: '#/components/responses/400'
+        '401':
+          $ref: '#/components/responses/401'
+        '422':
```
[the remark this answers is not in the document]
Comment (x1m3) [reply api/api.yaml:1323]: Nope. I will remove, sorry

File: internal/core/services/payment.go [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+			Amount:         chainConfig.Amount.String(),
+			ExpirationDate: fmt.Sprint(expirationTime.Format(time.RFC3339)),
+			Metadata:       metadata,
+			Recipient:      chainConfig.Recipient.String(),
+			Features:       setting.PaymentOption.Features,
+			TokenAddress:   setting.PaymentOption.ContractAddress.String(),
+			Proof:          paymentProof(&setting, signature, signerAddress),
+		}, nil
+
+	case protocol.Iden3PaymentRequestCryptoV1Type:
```
[the remark this answers is not in the document]
Comment (x1m3) [reply internal/core/services/payment.go]: Removed

**Review by demonsh — APPROVED** [review]

**Pull request merged**